cpp/include/cudf/functions.h

```diff
@@ -796,6 +796,36 @@ gdf_error gdf_min(gdf_column *col, void *dev_result, gdf_size_type dev_result_si
  * --------------------------------------------------------------------------*/
 gdf_error gdf_max(gdf_column *col, void *dev_result, gdf_size_type dev_result_size);
 
+/* --------------------------------------------------------------------------*
+ * @brief  Computes the rolling window function of the values in a column. This
```

Comment:
This is too long for @brief. You should use a 1-2 line comment for @brief, then leave a blank line, and then the more detailed description. Whether or not the API matches Pandas should be in the detailed description, not brief. Brief should just say what it does in one short sentence.